influxdb/_dataframe_client.py

```diff
@@ -131,25 +132,26 @@ def write_points(self,
                 protocol=protocol)
             return True
 
-    def query(self, query, chunked=False, database=None):
+    def query(self, query, dropna=True, **kwargs):
         """
         Quering data into a DataFrame.
 
-        :param chunked: [Optional, default=False] True if the data shall be
-            retrieved in chunks, False otherwise.
+        :param query: the actual query string
+        :param dropna: drop columns where all values are missing
+        :param **kwargs: additional parameters for ``InfluxDBClient.query``
 
         """
-        results = super(DataFrameClient, self).query(query, database=database)
+        results = super(DataFrameClient, self).query(query, **kwargs)
```

Comment:
Best to use named keyword arguments rather than `kwargs` magic. Any arguments passed to dataframe client not present in the super class function will cause a run time stack trace.